src/main/java/org/dependencytrack/event/kafka/processor/MirrorVulnerabilityProcessor.java

```diff
@@ -62,6 +62,12 @@ else if(mirrorSource.equals("NVD"))
                             default -> vulnerabilityAlias.setOsvId(cycloneVuln.getId());
                         }
                     }
+                    if (mirrorSource.equals("NVD")) {
+                        switch (advisorySource) {
+                            case GITHUB -> vulnerabilityAlias.setGhsaId(cycloneVuln.getId());
```

Comment:
Does the NVD report any aliases? I have to admit I did not check their new API, but I'd assume they only ever report CVEs without aliases.